artifactory/repositories.go

```diff
@@ -97,65 +97,65 @@ func (l LocalRepository) String() string {
+	VcsGitDownloadUrl                 *string                 `json:"vcsGitDownloadUrl,omitempty" xml:"vcs>git>downloadUrl,omitempty"`
+	BypassHeadRequests                *bool                   `json:"bypassHeadRequests,omitempty" xml:"bypassHeadRequests,omitempty"`
+	ClientTLSCertificate              *string                 `json:"clientTlsCertificate,omitempty" xml:"clientTlsCertificate,omitempty"`
+	ExternalDependenciesEnabled       *bool                   `json:"externalDependenciesEnabled,omitempty" xml:"externalDependencies>enabled,omitempty"`
+	ExternalDependenciesPatterns      *[]string               `json:"externalDependenciesPatterns,omitempty" xml:"externalDependencies>patterns>pattern,omitempty"`
+	DownloadRedirect                  *bool                   `json:"downloadRedirect,omitempty" xml:"downloadRedirect>enabled,omitempty"`
+	FeedContextPath                   *string                 `json:"feedContextPath,omitempty" xml:"nuget>feedContextPath,omitempty"`
+	DownloadContextPath               *string                 `json:"downloadContextPath,omitempty" xml:"nuget>downloadContextPath,omitempty"`
+	V3FeedUrl                         *string                 `json:"v3FeedUrl,omitempty" xml:"nuget>v3FeedUrl,omitempty"`
+	XrayIndex                         *bool                   `json:"xrayIndex,omitempty" xml:"xray>enabled,omitempty"`
+	ListRemoteFolderItems             *bool                   `json:"listRemoteFolderItems,omitempty" xml:"listRemoteFolderItems,omitempty"`
+	EnableTokenAuthentication         *bool                   `json:"enableTokenAuthentication,omitempty" xml:"enableTokenAuthentication,omitempty"`
+	ContentSynchronisation            *ContentSynchronisation `json:"contentSynchronisation,omitempty" xml:"contentSynchronisation,omitempty"`
+	BlockPushingSchema1               *bool                   `json:"blockPushingSchema1,omitempty" xml:"blockPushingSchema1,omitempty"`
+	QueryParams                       *string                 `json:"queryParams,omitempty" xml:"queryParams,omitempty"`
+	PropagateQueryParams              *bool                   `json:"propagateQueryParams,omitempty" xml:"propagateQueryParams,omitempty"`
 }
 
 // ContentSynchronisation represents smart remote repository configuration
 type ContentSynchronisation struct {
-	Enabled    *bool `json:"enabled,omitempty"`
+	Enabled    *bool `json:"enabled,omitempty" xml:"enabled,omitempty"`
 	Properties *struct {
-		Enabled *bool `json:"enabled,omitempty"`
-	} `json:"properties,omitempty"`
+		Enabled *bool `json:"enabled,omitempty" xml:"enabled,omitempty"`
+	} `json:"properties,omitempty" xml:"properties,omitempty"`
 	Statistics *struct {
-		Enabled *bool `json:"enabled,omitempty"`
-	} `json:"statistics,omitempty"`
+		Enabled *bool `json:"enabled,omitempty" xml:"enabled,omitempty"`
+	} `json:"statistics,omitempty" xml:"statistics,omitempty"`
 	Source *struct {
```

Comment:
What are your thoughts on the anonymous structs here?

Do you think there is any value in pulling these out into defined structs?

I'm fine to leave them if that is the preference of others 👍